WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameView.swift

```diff
@@ -6,7 +6,7 @@ import WordPressShared
 class SiteNameView: UIView {
 
     private var siteVerticalName: String
-    private let onContinue: (String?) -> Void
+    let onContinue: (String?) -> Void
```

Comment:
Access control was tweaked for this closure so that we could call from tests. I'm certainly open to better suggestions in this PR. Thanks!